src/hostapi/jack/pa_jack.c

```diff
@@ -57,6 +57,7 @@
 #include <errno.h>  /* EBUSY */
 #include <signal.h> /* sig_atomic_t */
 #include <math.h>
+#include <pthread.h>
```

Comment:
The PR says `regex.h`, yet this adds `pthread.h`, is this correct?

Reply:
Huh, I guess I messed up the commit message.